service/src/main/kotlin/vdi/model/DBPlatform.kt

```diff
@@ -1,6 +1,16 @@
 package vdi.model
 
+import org.veupathdb.vdi.lib.common.util.or
+
 enum class DBPlatform(val value: String) {
   Oracle("Oracle"),
-  Postgres("Postgres"),
+  Postgres("Postgresql");
+
+  companion object {
+    fun fromPlatformString(platformString: String?): DBPlatform {
+      return values().asSequence()
+        .find { platformString.equals(other = it.value, ignoreCase = true) }
+        .or { Oracle } // Default to Oracle
```

Comment:
_Might_ be better to use an elvis operator over a custom mixin function?  Doesn't really matter since `or` is inlined anyway.